src/composables/sdk.ts

```diff
@@ -15,26 +14,16 @@ interface UseSdkOptions {
  * For now it works as an abstraction layer.
  */
 export function useSdk({ store }: UseSdkOptions) {
-  const sdk = computed<ISdk | undefined>(() => store.getters['sdkPlugin/sdk']);
-  const isSdkReady = computed(() => !!sdk.value);
+  const sdk = computed<ISuperHeroAeSdk>(() => store.getters['sdkPlugin/sdk']);
 
   const isNodeConnecting = computed<boolean>(() => store.state.sdkPlugin.isNodeConnecting);
   const isNodeReady = computed<boolean>(() => store.state.sdkPlugin.isNodeReady);
   const isNodeError = computed<boolean>(() => store.state.sdkPlugin.isNodeError);
 
-  /**
-   * Get the SDK instance. For now the SDK state is asynchronous.
-   * TODO: With the new SDK version this probably could be replaced with a computed prop.
-   */
-  async function getSdk(): Promise<ISdk> {
-    return watchUntilTruthy(() => sdk.value);
-  }
-
```

Comment:
I suggest to not remove this function because when I was creating it the goal for me was to lazy initialize the SDK only when it's needed. So instead of the constant need of checking if the SDK is ready on the components level, this function would take care of this.